src/diffusers/models/unet_2d.py

```diff
@@ -245,4 +245,4 @@ def forward(
         if not return_dict:
             return (sample,)
 
-        return UNet2DOutput(sample=sample)
+        return sample
```

Comment:
We can't change this, as this will break our pytorch API and backward compatibility :sweat_smile: